src/System.IO.Pipes/tests/NamedPipeTests/NamedPipeTest.Simple.cs

```diff
@@ -365,7 +365,6 @@ public void WaitForConnectionOnAlreadyConnectedServer_Throws_InvalidOperationExc
             }
         }
 
-        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsSubsystemForLinux))] // https://github.com/Microsoft/BashOnWindows/issues/1011
         [PlatformSpecific(TestPlatforms.AnyUnix)] // Unix implemented on sockets, where disposal information doesn't propagate
```

Comment:
This should be replaced with [Fact] attribute, otherwise these tests won't be discovered by xunit..

Reply:
oops...